README.md

```diff
@@ -153,41 +151,6 @@ contentDirectories: |
     - target/prism.deb
 ```
 
-## Example - sources
```

Comment:
Is it worth preserving something like this but for `contentDirectories` instead? I've had word-of-mouth feedback from users that this section was really helpful to understand the bundling behaviour. `contentDirectories` behaves similarly so I suspect some kind of example is still valuable.